### ISSUE TITLE
Suggestion: Rustic Compatibility

### ISSUE THREAD

**Issue by MilchLos** [GrowthcraftCE/Growthcraft-1.11#58]
Rustic is a mod which shares many similarities with Growthcraft so i want to ask whether there will be some compatiblity with this mod.
For example both mods add Grapes to the world which can be made into Grape Wine (brewing in general)

Link to Rustic:
https://www.curseforge.com/minecraft/mc-mods/rustic

**Comment by Alatyami** [comment]
Yes, once the port is finished, we'll be looking into compatibility.
